tests/redis-trib.rb

```diff
@@ -0,0 +1,1402 @@
+#!/usr/bin/env ruby
```

Comment:
@etng Please explain the reason for redis-trib.rb to be included and why not use the built in one that is alreay in redis?

If you have made any modifications to this file, i would really like you to do this instead. First commit, add the file from redis source code and include from what commit hash it was taken. Second commit, include all changes you have done to this file (if any). I can't accept this unless i get that :]